### ISSUE TITLE
`install` command options

### ISSUE THREAD

**Issue by chouseknecht** [ansible/mazer#97]
Noticed that `mazer install --help` includes the following things... 

This one is confusing. What does it do, and should it be removed?

```
 -g, --keep-scm-meta

Use tar instead of the scm archive option when packaging the role
```

The following conflicts with `--content-path`, and should be removed:

```
-p ROLES_PATH, --roles-path=ROLES_PATH

The path to the directory containing your roles. The default is the roles_path configured in your
ansible.cfgfile (/etc/ansible/roles if not configured)
```

We decided to NOT allow installing individual content types from a package, so this one should go away too:

```
-t CONTENT_TYPE, --type=CONTENT_TYPE

A type of Galaxy Content to install: role, module, etc
```

**Comment by alikins** [comment]
See https://github.com/ansible/mazer/pull/98

**Comment by alikins** [comment]
Fixed in #98
